index.js

```diff
@@ -5,7 +5,7 @@ const { normalizePath } = require("vite");
 module.exports = function eslintPlugin(options = {}) {
   const {
     eslintOptions = {},
-    shouldLint = (path) => path.match(/\/src\/.*\.[jt]sx?$/),
+    shouldLint = (path) => path.match(/\/src\/.*\.m?[jt]sx?$/),
```

Comment:
to kill 2 birds with one stone (and issue #4), could we also try to support vue with sth like that: 
```suggestion
    shouldLint = (path) => path.match(/\/src\/.*\.(vue|m?[jt]sx?)$/),
```

Reply:
This should also avoid issue with vue virtual files:
```suggestion
    shouldLint = (path) => path.match(/\/src\/[^?]*\.(vue|m?[jt]sx?)$/),
```

Reply:
Big fan of this, good idea @pierroz127 !